spec/javascripts/jasmine_spec.rb

```diff
@@ -1,16 +1,26 @@
 require 'spec_helper'
 
+# Run the jasmine tests by running the jasmine:ci rake command and capturing the output
+#   The spec will pass if any jasmine tests pass
+#
+#   If you have a jasmine syntax error the test will fail since there will not be zero failures
+#
+#   If you add a new jasmine test by adding a file to spec/javascript/*spec.js* make sure the
+#   number of test run increments or you may have a syntax error inside your jasmine test
+#
 describe "Jasmine" do
   it "expects all jasmine tests to pass" do
     load_rake_environment ["#{jasmine_path}/lib/jasmine/tasks/jasmine.rake"]
     jasmine_out = run_task 'jasmine:ci'
     unless jasmine_out.include?  "0 failures"
-      puts "************************  Jasmine Output *************\n\n"
+      puts "\n\n************************  Jasmine Output *************"
       puts jasmine_out
       puts "************************  Jasmine Output *************\n\n"
+    else
+      puts "#{jasmine_out.match(/\n(.+) specs/)[1]} jasmine specs run"
```

Comment:
@mjgiarlo That only works if you have one test per file.  You can have many tests in one file.
I could change you line to say .. In <N> spec files